### PR TITLE
Sync dev branch with main

### PR DESCRIPTION
**Pull request by shinevue**
I am going to sync dev branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,8 @@
 
 ## Contributor
 
+#### [Shine](https://github.com/shine)
```
Comment (mcyandex) [inline README.md:56]: #### [Shine](https://github.com/shine)
change to
#### [Shine](https://github.com/shinevue)

File: README.md [hunk 2]
```diff
@@ -53,7 +53,8 @@
...
 #### [Truestar]
-#### [Shine](http://github.com/shine)
 #### [Yonex]
```
Comment (mcyandex) [inline README.md:58]: change to
#### [Yonex](https://github.com/mcyandex)

**Review by mcyandex — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shinevue) [reply README.md:56]: I got it.
Thank you for your review.

File: README.md (hunk 2 above)
Comment (shinevue) [reply README.md]: I already fixed it.

**Review by mcyandex — APPROVED** [review]

**Review by oleh1010 — APPROVED** [review]
I agree

**Pull request merged**